doc/frameworks/djl/using_djl.rst

```diff
@@ -0,0 +1,188 @@
+#######################################
+Use DJL with the SageMaker Python SDK
+#######################################
+
+With the SageMaker Python SDK, you can use Deep Java Library to host models on Amazon SageMaker.
+
+`Deep Java Library (DJL) Serving <https://docs.djl.ai/docs/serving/index.html>`_ is a high performance universal stand-alone model serving solution powered by `DJL <https://docs.djl.ai/index.html>`_.
+DJL Serving supports loading models trained with a variety of different frameworks. With the SageMaker Python SDK you can
+use DJL Serving to host large models using backends like DeepSpeed and HuggingFace Accelerate.
+
+For information about supported versions of DJL Serving, see the `AWS documentation <https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-images.html>`_.
+We recommend that you use the latest supported version because that's where we focus our development efforts.
+
+For general information about using the SageMaker Python SDK, see :ref:`overview:Using the SageMaker Python SDK`.
+
+.. contents::
+
+*******************
+Deploy DJL models
+*******************
+
+With the SageMaker Python SDK, you can use DJL Serving to host models that have been saved in the HuggingFace pretrained format.
+These can either be models you have trained/fine-tuned yourself, or models available publicly from the HuggingFace Hub.
+DJL Serving in the SageMaker Python SDK supports hosting models for the popular HuggingFace NLP tasks, as well as Stable Diffusion.
+
+You can either deploy your model using DeepSpeed or HuggingFace Accelerate, or let DJL Serving determine the best backend based on your model architecture and configuration.
+
+.. code:: python
+
+    # Create a DJL Model, backend is chosen automatically
+    djl_model = DJLModel(
+        "s3://my_bucket/my_saved_model_artifacts/",
+        "my_sagemaker_role",
+        djl_version="0.21.0",
```

Comment:
We should remove this line in example. Use latest version is recommended way

Reply:
Will remove `djl_version` from the example snippets